server/index.js

```diff
@@ -14,8 +14,8 @@ const webpackMiddleware = require('webpack-dev-middleware');
 const webpackHotMiddleware = require('webpack-hot-middleware');
 const config = require('../config/webpack.config.js');
 
-const Shopify = require('shopify-api-node');
```

Comment:
I don't see where this moved to. This constant is referenced in `registerWebhook`. It could be named `ShopifyAPI` to avoid confusion